<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsClosing</class>
 <widget class="QWidget" name="SettingsClosing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="WorldCloseBehavior">
     <property name="title">
      <string>Worlds</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="WorldCloseBehavior_Save">
        <property name="text">
         <string>Save worlds automatically</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="WorldCloseBehavior_Confirm">
        <property name="text">
         <string>Confirm before closing unsaved worlds</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="WorldCloseBehavior_Discard">
        <property name="text">
         <string>Close unsaved worlds without confirmation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ConfirmQuit_group">
     <property name="title">
      <string>SmushClient</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="ConfirmQuit">
        <property name="text">
         <string>Confirm before closing SmushClient</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ConfirmQuit_false">
        <property name="text">
         <string>Close SmushClient without confirmation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>187</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
